core/eolearn/core/eoexecution.py

```diff
@@ -201,6 +201,8 @@ def _execute_workflow(self, process_args):
             if return_results:
                 stats[self.RESULTS] = results
 
+        except KeyboardInterrupt:
+            raise KeyboardInterrupt()
```

Comment:
Comments:

- When you raise an exception without a custom message you don't have to initialize it. So instead of `raise KeyboardInterrupt()` you can do `raise KeyboardInterrupt`. It is a Python thing...
- Perhaps we want to propagate an entire stack trace to inform users where exactly in the code execution was interrupted. That can be achieved like this:

```Python
        except KeyboardInterrupt as exception:
            raise KeyboardInterrupt from exception
```

Reply:
Added tests to check for `KeyboardInterrupt` propagation